katana/data/data_base.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+
+class BaseData():
+    """Base class for interacting with different data sourses
+    """
+
+    fmt_date = '%Y%m%d'
```

Comment:
Since I consider this a constant I would capitalize it. [Flake8 recommends](https://www.python.org/dev/peps/pep-0008/#constants) this and I also prefer this since I can immediately differentiate between a local variable and something that was defined outside of a scope

Reply:
Made it also more self descriptive so I didn't have to add a comment